manifests/charts/istio-control/istio-discovery/templates/mutatingwebhook.yaml

```diff
@@ -37,10 +37,12 @@ webhooks:
     namespaceSelector:
 {{- if .Values.sidecarInjectorWebhook.enableNamespacesByDefault }}
       matchExpressions:
+      {{- if .Values.sidecarInjectorWebhook.excludeNamespaces }}
       - key: name
         operator: NotIn
         values:
-        - {{ .Release.Namespace }}
+        {{ toYaml .Values.sidecarInjectorWebhook.excludeNamespaces | indent 8 | trim }}
```

Comment:
isn't this a breaking change? We no longer will exclude the release namespace which may cause users istio control plane pods to be injected on upgrade and break?

Reply:
Looks like some of the components have the annotation, but not the labels. I can add the labels and annotation of no-inject where necessary (like cni, gateways, istiod etc)